scripts/buildNumberIncrement.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+INFO_PLIST_PATH=$1
+BUILD_NUMBER=$2
+
+/usr/libexec/PlistBuddy -c "Set :CFBundleVersion $BUILD_NUMBER" "$INFO_PLIST_PATH"
```

Comment:
Although is quite unlikely to happen, there could be already a version with the new build number uploaded to Hockey (you uploaded it manually, you replay the Jenkins job...). 
But I'm fine with this.

Reply:
This is something we talked about with Stephan. Previously, I used your script as-is. This means that I was taking the build number from Hockey and incrementing it with one.

But Stephan pointed out that it might be better to use the build number from Jenkins so they match. I mostly agree with his arguments so I changed it.

I think the worst case scenario here is that a build with the same build number is uploaded and it doesn't sort on top of other versions so it doesn't become the default version for download. But it's an edge case and I wouldn't worry too much.

Reply:
also, it's possible to set the jenkins build number manually in case someone does funny things with jenkins.

Reply:
I didn't know about BUILD_NUMBER environment variable 👍